doc/glm/meaning_difference.rst

```diff
@@ -0,0 +1,97 @@
+        * Second level: group level analysis
+    * FSL has a different meaning
+        * First level: analyze one run for one subject
+        * Second level: analyze across runs for one subject
+        * Third level: group level analysis
+
+These differences can be put in the table below:
+
++--------------------------------+----------------+------------------+---------------+
+| Analyze                        | nilearn        | SPM              | FSL           |
++================================+================+==================+===============+
+| One run, one subject           | First level    | First level      | First level   |
++--------------------------------+----------------+------------------+---------------+
+| More than one run, one subject | First level    | First level      | Second level  |
++--------------------------------+----------------+------------------+---------------+
+| More than or equal to one run, | Second level   | Second level     | Third level   |
+| more than one subject          |                |                  |               |
++--------------------------------+----------------+------------------+---------------+
+
+*Table showing the differences of the meaning for level of models between toolboxes\/libraries*
+
+Statistical Parametric Mapping (SPM)
+====================================
+
+SPM uses the same notation as Nilearn for analysis levels,
+with a note that in SPM terminology ``session`` refers to an imaging run,
+and within a run there could be multiple conditions (for example congruent and incongruent).
+`SPM`_ provides `tutorials`_ and documentation, including `lectures`_,
+to help users analyze their own fMRI data with the meaning of analysis levels being as follows:
+
+- `First level analysis in SPM`_: Analyze across runs for a subject
+  (meaning more than one run of one subject)
+- `Second level analysis in SPM`_: Analyze across several subjects
+  (meaning more than one subject with one or more run per subject).
+  This is also known as **group level analysis** which test
+  if the average estimate across subjects is statistically significant.
+
+.. admonition:: Fixed effects analyses
+    :class: hint
+
+    One important difference between SPM and Nilearn,
+    is that the typical first level workflow in SPM
+    will create a single design matrix for all runs
+    and thus run a single model at the subject level
+    (see this `example in Andy's tutorial <First level analysis in SPM>`_).
+    Nilearn will instead create one design matrix per run,
+    and run one model per run
+    (see for example :ref:`this report with 2 runs <two_runs_glm>`).
+    In Nilearn, to compute summary statistics across runs
+    you can use the method :meth:`~nilearn.glm.first_level.FirstLevelModel.compute_contrast`
+    or the function :func:`~nilearn.glm.compute_fixed_effects`.
+
+    .. seealso::
+
+        :ref:`sphx_glr_auto_examples_04_glm_first_level_plot_two_runs_model.py`
```

Comment:
@bthirion 
added this admonition regarding fixed effect analysis: does it make sense to you?

Reply:
That's very clear. Thx